.github/workflows/main.yml

```diff
@@ -119,13 +119,13 @@
   "repos_with_pages":
     "name": "Set up gh-pages branch"
     "needs":
+    - "strimzi"
```

Comment:
I wonder why the order changed.

Reply:
This is after I run `make configure` on a fresh checkout too.

```
Jsonnet commandline interpreter v0.17.0
```

Reply:
This seems to be dependent on the ordering of the libs passed into the command line, so:

```
IMPORTS=$(shell find libs -name config.jsonnet | xargs -I {} echo "(import '{}'),")
```

Reply:
I could not reproduce this, are you on MacOS perhaps?

Reply:
> I could not reproduce this, are you on MacOS perhaps?

Nope, arch, zsh with GNU find.